bin/oneoff/upload_non_en_videos.rb

```diff
@@ -0,0 +1,148 @@
+# This script is used to batch upload non-EN videos, with the assumption that
+# we will frequently get these in batches. They will be uploaded to YouTube
+# and S3 as well as added to videos.csv.
+
+require 'google/apis'
+require 'google/apis/youtube_v3'
+require 'google/api_client/client_secrets.rb'
+require 'googleauth'
+require 'googleauth/stores/file_token_store'
+require 'csv'
+require 'fileutils'
+require 'rubygems'
+require 'csv'
+require 'optparse'
+
+# Switch this to false when ready to run
+DRY_RUN = true
+
+APPLICATION_NAME = 'Dubbed Video Batch Upload'
+
+# This is the URI so that we can authorize from a browser. Don't change this.
+REDIRECT_URI = 'urn:ietf:wg:oauth:2.0:oob'
+
+SCOPE = Google::Apis::YoutubeV3::AUTH_YOUTUBE_UPLOAD
+
+def parse_options(args)
+  options = {}
+  opt_parser = OptionParser.new do |opts|
+    opts.banner = "Usage: upload_non_en_videos [options]"
+
+    opts.on("-c", "--client_secrets CLIENT_SECRETS_PATH", "Location of client_secrets.json file") do |client_secrets_path|
+      options[:client_secrets_path] = client_secrets_path
+    end
+
+    opts.on("-m", "--map_file MAP_CSV", "Location of csv file mapping filenames to video keys. Must have columns titled 'Folder', 'Partner filename', 'Key', and 'Video Title'") do |map_path|
```

Comment:
Does capitalization matter? Might want note that and/or make all of these lowercase just so its easier to make sure the case is correct.